github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-periodics.yaml

```diff
@@ -113,3 +113,61 @@ periodics:
     - name: gcs
       secret:
         secretName: gcs
+- name: periodic-containerized-data-importer-push-nightly-ARM64
+  cron: "2 3 * * *"
+  decorate: true
+  annotations:
+    testgrid-create-test-group: "false"
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+    preset-kubevirtci-quay-credential: "true"
+  extra_refs:
+    - org: kubevirt
+      repo: containerized-data-importer
+      base_ref: main
+      work_dir: true
+  cluster: prow-workloads
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+    - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/gcs/service-account.json
+      - name: DOCKER_PREFIX
+        value: quay.io/containerized-data-importer
+      - name: BUILD_ARCH
+        value: crossbuild-aarch64
+      command:
+        - "/usr/local/bin/runner.sh"
+        - "/bin/sh"
+        - "-c"
+        - |
+          set -e
+          build_date="$(date +%Y%m%d)"
+          cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io &&
+          export DOCKER_TAG="${build_date}_$(git show -s --format=%h)-arm64" make bazel-push-images
+          bucket_dir="kubevirt-prow/devel/nightly/release/kubevirt/containerized-data-importer/${build_date}"
+          gsutil cp ./_out/manifests/release/cdi-operator.yaml gs://$bucket_dir/cdi-operator-arm64.yaml
+          gsutil cp ./_out/manifests/release/cdi-cr.yaml gs://$bucket_dir/cdi-cr-arm64.yaml
```

Comment:
Could you please move this code to a script in `kubevirt/containerized-data-importer`? This way, if something needs to be changed in the future you can just update it there without having to change the job definition.

Reply:
I copied the gsutil stuff directly from the KubeVirt job. I can make it a script no problem its the push of the yaml to the gcs bucket. But I would suggest asking KubeVirt guys do the same.

Reply:
Also I don't think we have gsutils in the normal build, so it kind of doesn't make sense to put that in a script.

Reply:
Sure, you don't need to have gsutils in the normal build, it will be used when running in a container created from the image used in the CI job. 

The goal of creating a CI script in the repo where the code lives is to separate concerns, if you need to change anything related to how the test is run you would not need to create a PR on project-infra, you would just need to change the test logic on CDI.

Old jobs still have lengthy and "domain-specific" scripts in their definitions, but for new ones we are trying to make them just call a script on the relevant repo that encapsulates all the build logic, does that make sense?